src/com/lilithsthrone/game/Game.java

```diff
@@ -1024,18 +1017,22 @@ private static NPC loadNPC(Document doc, Element e, String className,
 				classMap.put(className, npcClass);
 				Method m = npcClass.getMethod("loadFromXML", Element.class, Document.class, CharacterImportSetting[].class);
 				loadFromXMLMethodMap.put(npcClass, m);
-				
+
 				Constructor<? extends NPC> declaredConstructor = npcClass.getDeclaredConstructor(boolean.class);
 				constructorMap.put(npcClass, declaredConstructor);
-				NPC npc = declaredConstructor.newInstance(true);
-				m.invoke(npc, e, doc, new CharacterImportSetting[] {});
-				return npc;
-			} else {
-				Constructor<? extends NPC> declaredConstructor = constructorMap.get(npcClass);
-				NPC npc = declaredConstructor.newInstance(true);
-				loadFromXMLMethodMap.get(npcClass).invoke(npc, e, doc, new CharacterImportSetting[] {});
-				return npc;
 			}
+			Constructor<? extends NPC> declaredConstructor = constructorMap.get(npcClass);
+			NPC npc = declaredConstructor.newInstance(true);
```

Comment:
This is throwing a NullPointerException, seemingly at random. I assume there's something going wrong with writing to the ConcurrentHashMaps, but I can't figure it out. :sweat_drops: 

Each time I try to load a game, it's a different class(es) that fails, and when it does, it's not for every NPC in that class, but just for one or two of them. Sometimes the loading works without any error being thrown at all. Sometimes it's as many as five (on my moderately-sized save file).